config/crd/bases/http.keda.sh_httpscaledobjects.yaml

```
@@ -60,6 +60,26 @@ spec:
           spec:
             description: HTTPScaledObjectSpec defines the desired state of HTTPScaledObject
             properties:
+              coldStartTimeoutFailoverRef:
+                description: (optional) The name of the fallback service to route
```

since we settled on the API terminology, can you please also adjust the code and comments to not say `fallback` but `failover` instead? It's going to be much more readable.
```suggestion
                description: (optional) The name of the failover service to route
```

---

Indeed, I've changed the code and comments accordingly 👍